### ISSUE TITLE
Is there any way to make a flatpak version for the 5.12 beta or make it so arch Linux users can install the beta?

### ISSUE THREAD

**Issue by Ares076** [lutris/lutris#4624]

**Comment by strycore** [comment]
the Flatpak is in a severely messed up situation right now 🪦

**Comment by mbaran5** [comment]
Is this related to why Discover is telling me to do a  "Refresh of version 0.5.11" and showing as an update?

**Comment by minimeee** [comment]
> the Flatpak is in a severely messed up situation right now

Can you give us some details? I've built a `0.5.12-beta` flatpak and it's working fine on my steam deck so far... Are there any known issues I haven't encountered yet? What are your plans regarding the Flatpak version?

**Comment by strycore** [comment]
Currently, the Flatpak build is failing. 
I don't know why.
My plan is to wait for someone else to figure it out.

**Comment by strycore** [comment]
If you have further questions regarding the Flatpak for 0.5.12, you can ask @A6GibKm

**Comment by minimeee** [comment]
> Currently, the Flatpak build is failing.

Are you referring to this error?
```
Applying patch webkit-version.patch
FB: Running: patch -p1 -i /srv/buildbot/worker/build-x86_64-1/build/webkit-version.patch
patching file lutris/gui/dialogs/webconnect_dialog.py
Hunk #1 FAILED at 3.
```

I had that too, but I've been able to fix it by:
* updating Lutris version to `10933ba9203a3f008646010005cdf19a6d97552c` and
* disabling `webkit-version.patch` in the manifest (and deleting the patch file)

Imho, the latter is not necessary anymore, since `webkit` version has been updated in Lutris main. But note that I chose the version right before https://github.com/lutris/lutris/commit/dcde7f528087ecab14d21941a0fde44ad9a19f2a (not sure why you reverted the webkit update commit, maybe because of a Flatpak build error?).

I can't create a PR, though, because of your most recent changes...
